source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -470,7 +471,7 @@ namespace PlayFab
 
         // Call SetBody
         {
-            auto requestUrl = GetUrl(requestTask);
+            const std::string& requestUrl = GetUrl(requestTask);
```

Comment:
>& [](start = 29, length = 1)

the other call to GetUrl was changed from a reference to value type but this one didn't change. Recommend doing another sweep